war/src/main/webapp/css/login.theme.css

```diff
@@ -0,0 +1,53 @@
+/*
+IE11 prevents of the usage of variable,
+but to show the color theme we left it here
+ */
+:root {
+    --main-bg-color-focus: #2171ce;
+    --main-bg-color: #4A90E2;
+    --danger-color: #C4000A;
+}
+html {
+    color: #4A4A4A;
+}
+.logo {
+    background-image: url('../images/jenkins.svg');
```

Comment:
We should not use SVG in Jenkins core since it is not fully supported by the HTMLUnit version used in this project. FYI https://github.com/jenkinsci/jenkins-test-harness/pull/95

Reply:
Cheers for the headsup. @jlong can you provide a png version?